var/spack/repos/builtin/packages/samurai/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Samurai(CMakePackage):
+    """Intervals coupled with algebra of set to handle adaptive
+    mesh refinement and operators on it"""
+
+    homepage = "https://github.com/hpc-maths/samurai"
+    git = "https://github.com/hpc-maths/samurai.git"
+
+    version("master", branch="master")
```

Comment:
```suggestion
    version("master", branch="master")
    version("0.21.1", tag="v0.21.1", commit="29798bd9592e5b80674c49080a242378a61878c3")
```

Let's add a stable version here too, just for folks who might want to use but not actively develop the package.